File: Mario2/mario.py
from cs50 import get_int

height = -1

while height<1 or height>8:
    height = get_int("Height: ")

if height >= 1 and height <= 8:

    for i in range(1,height + 1):
        spaces = height - i
        hashes = i
        print(" "*spaces+"#"*hashes)

        i = i + 1

File: Mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{   
    // initializes height
    int height = 0;

    // checks if user input is valid
    do
    {

        height = get_int("Height:");
    
    }
    while (height < 1 || height > 8);

    //creates rows
    for (int i = 0; i < height; i++)
    {
        //create columns
        for (int j = 1; j <= height; j++)
        {
            if (j >= height - i)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
            
        }
        printf("\n");
    }
}
